youths/notifications.py

```diff
@@ -0,0 +1,9 @@
+from django_ilmoitin.dummy_context import dummy_context
+from django_ilmoitin.registry import notifications
+
+notifications.register(
```

Comment:
could you do these as enums. [example](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/enums.py#L5)

Reply:
Sure, was supposed to but forgot. Thanks for 👁 it